components/grid/toolbar.md

```diff
@@ -22,17 +22,18 @@ The [Blazor Grid](https://demos.telerik.com/blazor-ui/grid/overview) provides se
 
 | Tool Name | Tool Tag | Description |
 | --- | --- | --- |
-| Add | `GridToolBarAddTool` | An add command that fires the [`OnAdd` event](slug:grid-editing-overview#events). |
-| Cancel | `GridToolBarCancelEditTool` | Cancels the changes for the selected row. [Row selection](slug:grid-selection-row) and [`Inline`](slug:grid-editing-inline) or [`Popup`](slug:grid-editing-popup) editing mode are required. |
+| Add | `GridToolBarAddTool` | An Add common button that fires the [`OnAdd` event](slug:grid-editing-overview#events). |
```

Comment:
- If "Add" is the button label, use bold (An **Add** command button). This comment applies to the rest of the descriptions in this table
- "common" -> "command"